.github/workflows/rust.yml

```
@@ -83,7 +83,7 @@ jobs:
       with:
         name: cargo-hack
         version: 0.5.16
-    - run: cargo-hack hack build --target wasm32-unknown-unknown --profile release
+    - run: for i in $(ls tests) ; do cargo-hack hack rustc --target wasm32-unknown-unknown --profile release --crate-type cdylib -p test_${i} ; done
```

Cargo-hack can do the lifting on assembling the list of packages to build, and if we exclude the problem package we can build more than the tests, but we can check that the SDK and other packages also build to wasm.
```suggestion
    - run: cargo-hack hack rustc --target wasm32-unknown-unknown --profile release --crate-type cdylib --workspace --exclude soroban-sdk-macros
```